src/server/config/config.js

```diff
@@ -17,7 +19,15 @@ config.CRISP_USER_HASH_SECRET = process.env.CRISP_USER_HASH_SECRET;
 config.PAYPAL_PDT_HOOK = process.env.PAYPAL_PDT_HOOK;
 config.LIKEPAY_HOOK = process.env.LIKEPAY_HOOK;
 
-config.EXTERNAL_URL = process.env.EXTERNAL_URL;
+config.LIKECOIN_CHAIN_API = IS_TESTNET
+  ? 'https://node.testnet.like.co'
+  : 'https://mainnet-node.like.co';
+config.LIKECOIN_API_BASE = IS_TESTNET
+  ? 'https://api.rinkeby.like.co'
+  : 'https://api.like.co';
+config.EXTERNAL_URL = IS_TESTNET
+  ? 'https://rinkeby.liker.land'
+  : 'https://liker.land';
```

Comment:
`EXTERNAL_URL` can be `http://localhost:3000` in dev, so please don't hardcode here